### PR TITLE
fix: tooltip: fixes tooltip visibility upon on mouse leave

### PR DESCRIPTION
**Pull request by mfazil-eightfold**
## JIRA TASK (Eightfold Employees Only):
https://eightfoldai.atlassian.net/browse/ENG-24280

## CHANGE TYPE:

-   [x] Bugfix Pull Request
-   [ ] Feature Pull Request

## TEST COVERAGE:

-   [ ] Tests for this change already exist
-   [ ] I have added unittests for this change

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -49,6 +49,7 @@
         ]
     },
     "dependencies": {
+        "@eightfold.ai/octuple": "^2.0.1",
```
Comment (dkilgore-eightfold) [inline package.json:52]: Is this necessary?

**Review by dkilgore-eightfold — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (mfazil-eightfold) [reply package.json:52]: This came up while installing packages. I'll remove it If it's not required.

Comment (ychhabra-eightfold) [reply package.json:52]: +1 remove this

File: yarn.lock [hunk 2]
```diff
@@ -2293,6 +2293,28 @@
   resolved "https://registry.yarnpkg.com/@discoveryjs/json-ext/-/json-ext-0.5.6.tgz#d5e0706cf8c6acd8c6032f8d54070af261bbbb2f"
   integrity sha512-ws57AidsDvREKrZKYffXddNkyaF14iHNHm8VQnZH6t99E8gczjNN0GpvcGny0imC80yQ0tHz1xVUKk/KFQSUyA==
 
+"@eightfold.ai/octuple@^2.0.1":
```
Comment (ychhabra-eightfold) [inline yarn.lock:2296]: this can be removed

**Review by ychhabra-eightfold — CHANGES_REQUESTED** [review]

**Pull request closed without merging**